packages/editor/README.md

```diff
@@ -114,6 +114,863 @@ Example:
 } )( window.wp.editor, window.React );
 ```
 
+## API
+
+<!-- START TOKEN(Autogenerated API docs) -->
+
+### AlignmentToolbar
+
+> **Deprecated** since 5.3, use `wp.blockEditor.AlignmentToolbar` instead.
```

Comment:
mmm! I expected these to not show up here at all. I wonder how some deprecated APIs don't show up at all in other packages.

Anyway, this is still good enough.

Reply:
Maybe we can omit the `deprecated` file from auto-generated API docs. Those old deprecated components aren't helpful for anyone.

Reply:
@Mamaduka Let's explore this separately, as it would require changes for the main docs script. My initial guess would be to add a new custom tag in jsodc to skip, but not sure.

Reply:
Sounds good to me 👍